### PR TITLE
Update Telemetry-2.0 TCK

### PR DESCRIPTION
**Pull request by Yas-Franco**
MpTelemetry 2.0 TCK with RC-1

### CODE REVIEW

**Comment by Yas-Franco** [comment]
#build
#libby

**Comment by Yas-Franco** [comment]
#build

**Comment by Yas-Franco** [comment]
The defects in the build are from an issue that should be fixed now. Running another build to be sure

#build

**Comment by Yas-Franco** [comment]
#build 
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

**Comment by Yas-Franco** [comment]
#libby

File: dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/publish/servers/Telemetry20LogsConfigTCKServer/bootstrap.properties [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+bootstrap.include=../testports.properties
+#com.ibm.ws.logging.trace.specification=*=info:com.ibm.ws.microprofile.rest*=all:org.apache.cxf.*=all:com.ibm.ws.jaxrs*=all
+com.ibm.ws.logging.max.files=1
+com.ibm.ws.logging.trace.specification=*=info:TELEMETRY=all
+otel.trace.http.disabled=true
+otel.sdk.disabled=false
+otel.metrics.exporter=none
+otel.traces.exporter=none
+otel.logs.exporter=logging
+log.file.path=${server.output.dir}/logs/messages.log
```
Comment (benjamin-confino) [inline dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/publish/servers/Telemetry20LogsConfigTCKServer/bootstrap.properties:22]: Is this for the TCK to know where to find and read the logs? If so I think it should be something like otel.tck.log.file.path

Comment (Yas-Franco) [reply dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/publish/servers/Telemetry20LogsConfigTCKServer/bootstrap.properties:22]: It shouldn't be an otel config property. It is a separate property for the TCK: 

https://github.com/eclipse/microprofile-telemetry/blob/main/tck/logs/src/main/java/org/eclipse/microprofile/telemetry/logs/tck/application/JulTest.java#L57

Comment (benjamin-confino) [reply dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/publish/servers/Telemetry20LogsConfigTCKServer/bootstrap.properties:22]: I mean that I think the TCK itself should use a property name like `otel.tck.log.file.path` to make sure it doesn't clash with a different thing also using `log.file.path`.

**Comment by benjamin-confino** [comment]
Approving this. The comment about the property name is really an issue to raise for the TCK and until that's done this PR is correct. 

Congrats on getting this in Yasmin :)

**Review by benjamin-confino — APPROVED** [review]

**Pull request merged**